### PR TITLE
Adjust Training Process

### LINKED ISSUE #15: Adjust Training Process
**Describe the enhancement**
Adjust the *epochs*, and *learning rate* parameters of the training process to ensure that the **validation loss** decreases more and to improve the **convergence speed** of the validation loss. This could also improve the accuracy of the model.

### PR DESCRIPTION
**Pull request by sagedemage**
Adjusted the *epochs*, and  *learning rate* parameters of the training process to ensure that the **validation loss** decreases more and to improve the **convergence speed** of the validation loss. This improved the accuracy of the model.

### CODE REVIEW

**Review by sagedemage — COMMENTED** [review]
LGTM!
